ambry-replication/src/main/java/com.github.ambry.replication/ReplicaThread.java

```diff
@@ -320,13 +323,16 @@ void replicate(List<List<RemoteReplicaInfo>> replicasToReplicate) {
       throws IOException, ReplicationException, InterruptedException {
 
     long exchangeMetadataStartTimeInMs = SystemTime.getInstance().milliseconds();
-    List<ExchangeMetadataResponse> exchangeMetadataResponseList = new ArrayList<ExchangeMetadataResponse>();
+    List<ExchangeMetadataResponse> exchangeMetadataResponseList = new ArrayList<>();
     if (replicasToReplicatePerNode.size() > 0) {
       try {
         DataNodeId remoteNode = replicasToReplicatePerNode.get(0).getReplicaId().getDataNodeId();
         ReplicaMetadataResponse response =
             getReplicaMetadataResponse(replicasToReplicatePerNode, connectedChannel, remoteNode);
         long startTimeInMs = SystemTime.getInstance().milliseconds();
+
+        Map<StoreKey, StoreKey> remoteToLocal = batchConvertReplicaMetadataResponseKeys(response);
```

Comment:
would prefer a more descriptive name here and in the function arguments - `remoteKeyToLocalKeyMap` or similar

Reply:
fixed